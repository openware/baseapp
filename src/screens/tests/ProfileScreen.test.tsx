import { shallow } from 'enzyme';
import React from 'react';
import { TestComponentWrapper } from 'lib/test';
import { ProfileScreen } from '../';

const setup = () =>
    shallow(
        <TestComponentWrapper>
            <ProfileScreen />
        </TestComponentWrapper>
    );

describe('ProfileScreen test', () => {
    it('should render', () => {
        const wrapper = setup().render();
        expect(wrapper).toMatchSnapshot();
    });
});
