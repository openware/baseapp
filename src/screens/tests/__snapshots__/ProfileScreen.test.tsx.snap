// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileScreen test should render 1`] = `ShallowWrapper {}`;
